src/components/CollectionsAndPalettes/common/RoundButton/RoundButton.tsx

```diff
@@ -1,45 +1,155 @@
 import * as React from "react";
 import cn from "classnames";
-import { AppContext, AppContextState } from "../../../../context/AppContext";
 import { Link } from "react-router-dom";
-import { COLLECTIONS_URL, PALETTES_URL } from "../../../../constants/urls";
+import {
+  COLLECTIONS_URL,
+  PALETTES_URL,
+  COLLECTIONS_EDIT_URL,
+  COLLECTIONS_INFO_URL,
+  COLLECTIONS_IMAGE_URL,
+} from "../../../../constants/urls";
 
 interface Props {
   buttons: any;
+  explore?: boolean;
```

Comment:
I think this RoundButton component is trying to do too many things, and it's getting a bit too difficult to follow.
I suggest we split it out somehow. Suggestion:
RoundButtons:
  -buttons: string[]
  -selectedButton: string
  -onButtonSelected: Function
  -buttonClassName?: string - if we want to change button styling from the outside
  -activeButtonClassName?: string
  -maybe other needed styling props
This one should only be for buttons like "Collections / Palettes" and the explore tags

Make a separate component for "image / info / edit" style buttons, as that is quite different in the implementation, but make it similar in regards to props: (i.e. get a list of buttons, the active one, and if needed some extra classNames).

For both of those, the parent component should be holding the selected button in its state, as the parent is supposed to either filter the content or change some stuff based on which button is selected.

For the most part, we should try to avoid adding more and more props to a component in order to implement a specific case. When that happens, usually it means that the component is not re-usable enough, and maybe we should look into moving some logic outside of it so that the parent can customize it enough.